backend_new/src/services/application-csv-export.service.ts

```diff
@@ -0,0 +1,653 @@
+      },
+      {
+        path: 'applicant.applicantWorkAddress.city',
+        label: `Primary Applicant Work City`,
+      },
+      {
+        path: 'applicant.applicantWorkAddress.state',
+        label: `Primary Applicant Work State`,
+      },
+      {
+        path: 'applicant.applicantWorkAddress.zipCode',
+        label: `Primary Applicant Work Zip Code`,
+      },
+      {
+        path: 'alternateContact.firstName',
+        label: 'Alternate Contact First Name',
+      },
+      {
+        path: 'alternateContact.middleName',
+        label: 'Alternate Contact Middle Name',
+      },
+      {
+        path: 'alternateContact.lastName',
+        label: 'Alternate Contact Last Name',
+      },
+      {
+        path: 'alternateContact.type',
+        label: 'Alternate Contact Type',
+      },
+      {
+        path: 'alternateContact.agency',
+        label: 'Alternate Contact Agency',
+      },
+      {
+        path: 'alternateContact.otherType',
+        label: 'Alternate Contact Other Type',
+      },
+      {
+        path: 'alternateContact.emailAddress',
+        label: 'Alternate Contact Email Address',
+      },
+      {
+        path: 'alternateContact.phoneNumber',
+        label: 'Alternate Contact Phone Number',
+      },
+      {
+        path: 'alternateContact.address.street',
+        label: `Alternate Contact Street`,
+      },
+      {
+        path: 'alternateContact.address.street2',
+        label: `Alternate Contact Street 2`,
+      },
+      {
+        path: 'alternateContact.address.city',
+        label: `Alternate Contact City`,
+      },
+      {
+        path: 'alternateContact.address.state',
+        label: `Alternate Contact State`,
+      },
+      {
+        path: 'alternateContact.address.zipCode',
+        label: `Alternate Contact Zip Code`,
+      },
+      {
+        path: 'income',
+        label: 'Income',
+      },
+      {
+        path: 'incomePeriod',
+        label: 'Income Period',
+        format: (val: string): string =>
+          val === 'perMonth' ? 'per month' : 'per year',
+      },
+      {
+        path: 'accessibilityMobility',
+        label: 'Accessibility Mobility',
+      },
+      {
+        path: 'accessibilityVision',
+        label: 'Accessibility Vision',
+      },
+      {
+        path: 'accessibilityHearing',
+        label: 'Accessibility Hearing',
+      },
+      {
+        path: 'householdExpectingChanges',
+        label: 'Expecting Household Changes',
+      },
+      {
+        path: 'householdStudent',
+        label: 'Household Includes Student or Member Nearing 18',
+      },
+      {
+        path: 'incomeVouchers',
+        label: 'Vouchers or Subsidies',
+      },
+      {
+        path: 'preferredUnitTypes',
+        label: 'Requested Unit Types',
+        format: (val: UnitType[]): string => {
+          return val
+            .map((unit) => this.unitTypeToReadable(unit.name))
+            .join(',');
```

Comment:
Same issue is happening here with the comma 
<img width="821" alt="image" src="https://github.com/bloom-housing/bloom/assets/42942267/26a99b0e-e875-44a9-9b7d-fd3efc688193">